packages/shared/src/lib/schema/opportunity.ts

```diff
@@ -12,27 +12,25 @@ const processSalaryValue = (val: unknown) => {
 };
 
 export const opportunityEditInfoSchema = z.object({
-  title: z.string().nonempty(labels.form.required).max(240),
-  tldr: z.string().nonempty(labels.form.required).max(480),
+  title: z.string().nonempty('Add a job title').max(240),
+  tldr: z.string().nonempty('Add a short description').max(480),
```

Comment:
Should also update API side same schemas, (we need common package 😆 ) just so it has same errors